hacks/delete-kind-local-shoot.sh

```diff
@@ -0,0 +1,2 @@
+kubectl -n garden-local annotate shoot local "confirmation.gardener.cloud/deletion=true" --overwrite
```

Comment:
Missing copyright notice.

Reply:
we won't be licensiny hacks/script with calling kubectl :) not IP there at all :)